plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -109,6 +111,15 @@ class DefaultDescriptorToDocumentableTranslator(
             )
         }
     }
+
+    override fun translateDescriptor(descriptor: ClassDescriptor, sourceSet: DokkaSourceSet): DClasslike {
```

Comment:
`translateClassDescriptor`? Likewise there is the same naming convention  for `visit`-methods.